From 3cb49e891432e0b32559812a70ce6124d36d6fe0 Mon Sep 17 00:00:00 2001
From: Captain Polliwog build <build@captainpolliwog.invalid>
Date: Thu, 17 Sep 2026 23:51:20 -0400
Subject: [PATCH 01/42] CMake Mac port: find Accelerate.framework
 case-sensitively

Builds on a case-sensitive file system (cross-compiling from Linux) cannot
find "accelerate".
---
 Source/WebCore/PlatformMac.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/WebCore/PlatformMac.cmake b/Source/WebCore/PlatformMac.cmake
index c035146f..87923737 100644
--- a/Source/WebCore/PlatformMac.cmake
+++ b/Source/WebCore/PlatformMac.cmake
@@ -7,7 +7,7 @@ set(WEBKITSYSTEMINTERFACE_LIBRARY libWebKitSystemInterfaceElCapitan.a)
 endif ()
 link_directories(../../WebKitLibraries)
 
-find_library(ACCELERATE_LIBRARY accelerate)
+find_library(ACCELERATE_LIBRARY Accelerate)
 find_library(APPLICATIONSERVICES_LIBRARY ApplicationServices)
 find_library(AVFOUNDATION_LIBRARY AVFoundation)
 find_library(AUDIOTOOLBOX_LIBRARY AudioToolbox)
-- 
2.50.1 (Apple Git-155)

